This ain't h2 (#1932)
httpwg/http-extensions: draft-ietf-httpbis-binary-message.md

transformation of entire messages, including the application of authenticated
encryption.

> The design of this format is a informed by the framing structure of HTTP/2

Reviewer: ```suggestion
The design of this format is informed by the framing structure of HTTP/2
```

## Request Control Data

> Request Control Data contains the method and target of a request message. That

Reviewer: I notice now the capitalization here is incosistent with the rest of the body. I'll let you pick how to resolve that.

## Response Control Data

> The control data for a request message includes a single field that includes the
> status code.  This field is encoded as a single variable length integer, not a
> decimal string.

Reviewer: For consistency with the request para (and fixing incorrect use of `request message` here)

```suggestion
Response Control Data contains the status code of a response message. It is
encoded as a single variable length integer, not a decimal string.
```

* framing of responses that depends on the corresponding request (such as HEAD)
  or the value of the status code (such as 204 or 304)

> This format shares some of these properties with HTTP/2 and HTTP/3.

Reviewer: This sentence now reads a bit strangely in combination with the list above. In simple reading it goes "this format lacks X, Y, Z that are defined for H2 and H3 ... this format shares some of the features we just said are lacking". 

Perhaps rephrase this as "some of these features are similarly not supported by HTTP/2 and HTTP/3, making the features unique to HTTP/1.1"?

> This format shares some of these properties with HTTP/2 and HTTP/3.

Reviewer: This sentence now reads a bit strangely in combination with the list above. In simple reading it goes "this format lacks X, Y, Z that are defined for H2 and H3 ... this format shares some of the features we just said are lacking". 

Perhaps rephrase this as "some of these features are similarly not supported by HTTP/2 and HTTP/3, making the features unique to HTTP/1.1"?

> This format shares some of these properties with HTTP/2 and HTTP/3.

Reviewer: This sentence now reads a bit strangely in combination with the list above. In simple reading it goes "this format lacks X, Y, Z that are defined for H2 and H3 ... this format shares some of the features we just said are lacking". 

Perhaps rephrase this as "some of these features are similarly not supported by HTTP/2 and HTTP/3, making the features unique to HTTP/1.1"?
